cli/tests/runner.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+
+mod common;
+
+#[test]
+fn test_no_abandoned_test_files() {
```

Comment:
Non-blocking: Perhaps `no_forgotten_test_files` (here and in the testutils function) is a little better?

I'm not completely happy with that name either, but we can always change the name later.